### DISCUSSION TITLE
Bonsai won't show video

### DISCUSSION THREAD

**Discussion by ZacariasBarreto** [bonsai-rx/bonsai#1782] [Q&A] [answered]
Even using just a filecapture node, and giving it a correct file name; after I click Start, the Bonsai window will say it is running, and I can see the CPU usage go up, although it doesn't show me the video. 

I don't know if this matters, but I am using Bonsai, whereas, when I was using Bonsai(x86), it showed the "An attempt was made to load a program with an incorrect format (Exception from HRESULT: 0x8007000B)" error, so I tried switching and it stopped showing that error.
Also, I am trying to use it with the PredictPose node from the Deeplabcut package

please help ! thank you

**Comment by bruno-f-cruz** [comment, marked as answer]
Heya
With a single FileCapture operator, did you try to double click the node while bonsai is running? What happens?

**Reply by ZacariasBarreto** [reply]
Hello! I tried this and, although some times it just says "Object reference not set to an instance of an object.", most of the time it works :D
